public/blocking.css

```diff
@@ -0,0 +1,103 @@
+html, body {
+    min-height: 100%;
```

Comment:
As seen in 14757d43f3c5b1eec87cb729365242914a11b06b, not setting `height` is not enough and makes the content align to the top of the page, instead of be vertically centered.